plugin.video.mlbtv/addon.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.mlbtv" name="MLB.TV®" version="2017.3.7" provider-name="eracknaphobia">
+  <requires>
+   <import addon="xbmc.python" version="2.25.0"/>   
+   <import addon="script.module.simplejson" version="3.3.0"/>  
+   <import addon="script.module.pytz" version="2014.2" />   
+   <import addon="script.module.beautifulsoup4" version="4.3.1" />
+   <import addon="script.module.pil" version="1.1.7"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="main.py">
+        <provides>video</provides>
+  </extension>  
+  <extension point="xbmc.addon.metadata">    
+    <platform>all</platform>
+    <summary lang="en">Watch every out-of-market regular season game in the office or on the go. The #1 LIVE Streaming Sports Service</summary>
```

Comment:
please change these to `en_GB` as it is the new default (only for these 3 localized strings not for the language tag)